app/router.js

```diff
@@ -34,8 +34,11 @@ export async function router(auth, id, payload, verbose) {
 
   for (const command of commands) {
     const result = command.enabled(config);
-    result.enabled
-      ? await command.run(authToken)
-      : await reportError(authToken, payload.issue.node_id, result.error);
+    await (result.enabled
+      ? Promise.all([
+          addReaction(authToken, payload.issue.comment.node_id, "THUMBS_UP"),
```

Comment:
Oh wait we are adding a comment for each commands recognized.

Perhaps we just do thumbs up whether it is enabled or not to show that we received the command.